### PR TITLE
Migrate l10n_fi_edicode 14 -> 15

### PR DESCRIPTION
**Pull request by aisopuro**
Changes are in last 3 commits of the diff.

Ping #47

### CODE REVIEW

**Comment by aisopuro** [comment]
Sorry for the pinging, but can @jarmokortetjarvi and @mlaitinen help me get this reviewed and merged? We'd love to have it fast if possible. Let me know if I can help get it done.

File: l10n_fi_edicode/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,7 @@
+This module allows users to configure Finnish eIinvoicing settings on your company and customers.
```
Comment (mlaitinen) [inline l10n_fi_edicode/readme/DESCRIPTION.rst]:
```suggestion
This module allows users to configure Finnish eInvoicing settings on your company and customers.
```

File: l10n_fi_edicode/readme/DESCRIPTION.rst [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+These settings include
+
+* Edicode: the indetifier used to identify a person or organization in eInvoicing services.
```
Comment (mlaitinen) [inline l10n_fi_edicode/readme/DESCRIPTION.rst]:
```suggestion
* Edicode: the identifier used to identify a person or organization in eInvoicing services.
```

**Review by mlaitinen — CHANGES_REQUESTED** [review]
Just a few typos, otherwise looks good. I started the runboat instance and tested the translations as well.

File: l10n_fi_edicode/readme/DESCRIPTION.rst (hunk 1 above)
Comment (aisopuro) [reply l10n_fi_edicode/readme/DESCRIPTION.rst]: Fixed

File: l10n_fi_edicode/readme/DESCRIPTION.rst (hunk 2 above)
Comment (aisopuro) [reply l10n_fi_edicode/readme/DESCRIPTION.rst]: Fixed

**Review by mlaitinen — APPROVED** [review]

**Pull request merged**